src/Resources/skeleton/registration/RegistrationController.tpl.php

```diff
@@ -7,11 +7,11 @@
 class <?= $class_name; ?> extends <?= $parent_class_name; ?><?= "\n" ?>
 {
 <?php if ($will_verify_email): ?>
-    private $emailVerifier;
+<?= $email_verifier_details->getPropertyStatement() ?>
```

Comment:
If there any advantage to having the `getPropertyStatement()` do the indentation and `;` for us? That seems like extra magic - and a little bit less readability here.